implementation/src/test/java/io/smallrye/health/checks/InetAddressHealthCheckTest.java

```diff
@@ -0,0 +1,27 @@
+package io.smallrye.health.checks;
+
+import org.eclipse.microprofile.health.HealthCheckResponse;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class InetAddressHealthCheckTest {
+
+    @Test
+    public void testInetAddressCheck() {
+        final InetAddressHealthCheck inetAddressHealthCheck = new InetAddressHealthCheck("127.0.0.1");
+        final HealthCheckResponse healthCheckResponse = inetAddressHealthCheck.call();
+
+        assertEquals("127.0.0.1", healthCheckResponse.getData().get().get("host"));
```

Comment:
I just realized that it's worth checking the default value for name is coming through in the response as well